src/controllers/__tests__/ticket.test.ts

```diff
@@ -106,21 +106,25 @@ describe("createTicket", () => {
 });
 
 describe("editTicket", () => {
-  let mockTicket: Ticket;
+  let mockedProject: Project;
 
   beforeAll(async () => {
-    mockTicket = await createTestTicket();
+    mockedProject = await createTestProject();
   });
 
   afterAll(async () => {
-    await mockTicket.remove();
+    await mockedProject.remove();
   });
 
   test("Passed appropriate body and query - it should successfully edit existing ticket", async () => {
     const mockName = "this is mocked name";
+    const mockedProjectId = mockedProject.id;
+    const mockedColumnId = mockedProject.columns[0].id;
+    const mockedTicketId = mockedProject.tickets[0].id;
+
     const res = await app.inject({
       method: "put",
-      url: "/tickets/1",
+      url: `/projects/${mockedProjectId}/columns/${mockedColumnId}/tickets/${mockedTicketId}`,
```

Comment:
These tests were failing with `404`. Turned out that it wasn't up to date with server.ts' change